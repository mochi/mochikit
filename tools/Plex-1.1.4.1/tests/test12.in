'Single-line text literal''Multi-
line text literal'