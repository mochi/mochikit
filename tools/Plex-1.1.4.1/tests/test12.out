test12.in   1   0 'Single-line text literal' "'Single-line text literal'"
test12.in   1  26 'Multi-
line text literal' "'Multi-\nline text literal'"
test12.in   2  18 None       ''
